vivo-project/VIVO
# [VIVO-1415] Add publication claiming from PubMed and CrossRef

**[VIVO-1415](https://jira.duraspace.org/browse/VIVO-1415)**: 

# What does this pull request do?
Implements the DOI and PMID claiming process from OpenVIVO

# What's new?
Adds the claiming interface - links to claim by DOI or PMID and the additional pages for entering IDs and confirming the links.

Most of the code is self-contained in the package org.vivoweb.webapp.createandlink;

Only small changes to the OpenVIVO code has been made, in order to move CSS classes to the appropriate theme files, add the claiming links into both the Wilma and Tenderfoot themes, and to make URIs conform to the standard VIVO format (as using the URIs generated in OpenVIVO would require further customisation of VIVO).

Also, there are contribution options, as the contribution ontology is not formally adopted into VIVO / we haven't considered how this applies to publications that aren't sourced via the claim interface, etc.

# How should this be tested?
Log in to VIVO with the ability to edit.
Go to a profile and verify the options for claiming are there.
Click on the claiming links - you should be claiming articles for the profile that you came from
Enter IDs, check that the publications are returned as expected, and confirming creates new publications and user links (note that if the DOI or PMID is already present, then the claim interface will return the existing publication from VIVO, not the external metadata).
Go to a co-author profile and claim the publication for them, verifying that the publication now links them as an author.

# Additional Notes:
This patch relies on the changes in this ticket / pull request:
https://jira.duraspace.org/browse/VIVO-1656
Vitro PR: https://github.com/vivo-project/Vitro/pull/99
VIVO PR: https://github.com/vivo-project/VIVO/pull/105

# Interested parties
@VIVO-project/vivo-committers
